Documentation/artifact-format.md

```diff
@@ -155,10 +155,7 @@ image file / package file / etc.), listed as bare file names. There may be one
 or multiple files listed.  For example:
 
 ```
-{ "files" : [
-  {"file": "core-image-minimal-201608110900.ext4"},
-  {"file": "core-image-base-201608110900.ext4"}
-]}
+{ "file" : ["core-image-minimal-201608110900.ext4", "core-image-base-201608110900.ext4"]}
```

Comment:
Minor: I think it should be `files` rather than `file` to signal that there can be multiple entries.